### ISSUE TITLE
StripeLink: Move billing email field to top

### ISSUE THREAD

**Issue by dpaun1985** [Automattic/woocommerce-payments#3842]
### Description

When Link payment method is enabled, on checkout page, the billing email field should be displayed the first input field in the form.

Reference: #3702

**Comment by dpaun1985** [comment]
blocked by #3802

**Comment by kalessil** [comment]
@mordeth, is this duplicate of https://github.com/Automattic/woocommerce-payments/issues/3801 ?

**Comment by mordeth** [comment]
> @mordeth, is this duplicate of #3801 ?

Yes, that one is a duplicate. Closing it.
